packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -64,7 +83,7 @@ module.exports = {
     // containing code from all our entry points, and the Webpack runtime.
     filename: 'static/js/bundle.js',
     // In development, we always serve from the root. This makes config easier.
```

Comment:
The change directly contradicts the comment above it. Please be sure to read the comments above the code you are changing, and keep them in sync.